payments/mobile-pay/other-features.md

```diff
@@ -94,7 +94,7 @@ Content-Type: application/json
 | {% icon check %} | └➔&nbsp;`intent`                | `string`     | `Authorization`                                                                                                                                                                                                                                           |
 | {% icon check %} | └➔&nbsp;`currency`              | `string`     | `NOK`, `SEK`, `DKK`, `USD` or `EUR`.                                                                                                                                                                                                                      |
 | {% icon check %} | └➔&nbsp;`prices`                | `object`     | The prices object.                                                                                                                                                                                                                                        |
-| {% icon check %} | └─➔&nbsp;`type`                 | `string`     | `Visa` (for card type Visa), `MC` (for card type Mastercard)                                                                                                                                                                                              |
+| {% icon check %} | └─➔&nbsp;`type`                 | `string`     | `Visa` (for card type Visa), `MC` (for card type Mastercard), `MobilePay` (for all card types).                                                                                                                                                                                              |
```

Comment:
The way I read this sentence, I have to send in `MobilePay` to have it show all types. Is this correct?

Reply:
If I understood this correctly, then yes (?) https://payex.slack.com/archives/CND0UNK2Q/p1594650704196700

Reply:
Ye alright